src/hdmf/common/alignedtable.py

```diff
@@ -257,9 +278,29 @@ def get(self, item, **kwargs):
                 return self.get_category(item).to_dataframe()
         elif isinstance(item, tuple):
             if len(item) == 2:
-                return self.get_category(item[0])[item[1]]
+                if isinstance(item[0], (int, np.integer)):
```

Comment:
Please add a description for this case in the function docstring, and optionally the error message on line 305.

Reply:
Ok, I have overhauled the docstring for ``AlignedDynamicTable.get`` and add further documentation to the body and error message.